### PR TITLE
Add crypto built-in to Akamai-EdgeWorkers

### PR DESCRIPTION
**Pull request by evan-hughes**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://techdocs.akamai.com/edgeworkers/docs/crypto
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,5 +56,8 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
+    },
+    "dependencies": {
+        "dtslint": "^4.2.1"
```
Comment (iisaduan) [inline package.json:61]: Please submit this change as a separate PR if this was intentional

**Review by iisaduan — CHANGES_REQUESTED** [review]

**Review by bmatthew — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (evan-hughes) [reply package.json]: Thanks @iisaduan, I've removed those extraneous changes.

**Comment by evan-hughes** [comment]
Ready to merge
